# The zlib path is not correctly handled for non-win32 systems
# through the ZLIB scons variable.
# Davide Vanzo - Vanderbilt University
diff -ru serf-1.3.9.orig/SConstruct serf-1.3.9/SConstruct
--- serf-1.3.9.orig/SConstruct	2018-10-26 11:05:46.410815739 -0500
+++ serf-1.3.9/SConstruct	2018-10-26 11:01:48.034815995 -0500
@@ -373,6 +373,8 @@
   env.Append(CPPPATH=['$OPENSSL/include'])
   env.Append(LIBPATH=['$OPENSSL/lib'])
 
+  env.Append(CPPPATH=['$ZLIB/include'])
+  env.Append(LIBPATH=['$ZLIB/lib'])
 
 # If build with gssapi, get its information and define SERF_HAVE_GSSAPI
 if gssapi and CALLOUT_OKAY:
